#! /bin/sh /usr/share/dpatch/dpatch-run
## 20-cyrusBuildConfig.dpatch by Peter Eisentraut <petere@debian.org>
##
## DP: Disables buildCyrusConfig function, because we don't want to provide the
## DP: necessary template files.  This unbreaks kolabconf.

@DPATCH@
diff -urNad libkolab-perl-5.8.7-20070420~/Kolab-Conf/Conf.pm.in libkolab-perl-5.8.7-20070420/Kolab-Conf/Conf.pm.in
--- libkolab-perl-5.8.7-20070420~/Kolab-Conf/Conf.pm.in	2007-01-06 22:36:20.000000000 +0100
+++ libkolab-perl-5.8.7-20070420/Kolab-Conf/Conf.pm.in	2007-07-27 19:29:00.000000000 +0200
@@ -434,6 +434,7 @@
 
 sub buildCyrusConfig
 {
+    return; # not used on Debian
     Kolab::log('T', 'Building Cyrus config', KOLAB_DEBUG);
 
     my $keytemplate = "$templatedir/cyrus.conf.template";
